front_end/html/summarize_logs.html

```diff
@@ -0,0 +1,158 @@
+
+<form method="post" action="/summarize_logs">
+
+<div class="columns">
+  <div class="column is-half">
+    <h3>CodeBuddy Summarized Logs</h3>
+    <div class="field is-grouped">
+      <div class="control">
+        <label class="label">Select File</label>
+        <div class="select is-info is-medium">
+          <select id="file_select" name="file_select">
+            <option selected disabled>Select File</option>
+            <option value="logs/summarized/HitsAnyUser.tsv.gz">HitsAnyUser</option>
+            <option value="logs/summarized/HitsPerUser.tsv.gz">HitsPerUser</option>
+            <option value="logs/summarized/LoadDuration.tsv.gz">LoadDuration</option>
+          </select>
+        </div>
+      </div>
+      <div class="control">
+        <label class="label">Filter Location</label>
```

Comment:
Let's call this "Page" or "Root directory" if it's referring to the first part of the URL